### PR TITLE
Added documentations for enabling hours of effort

### PR DESCRIPTION
**Pull request by BenjiLee**
https://openedx.atlassian.net/browse/LEARNER-3836

### CODE REVIEW

File: docs/feature_toggling.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Feature Toggling
```
Comment (amangano-edx) [inline docs/feature_toggling.rst]: Did you change anything in this file? Or did you just move all the existing text to a new location?

File: docs/features.rst [hunk 2]
```diff
@@ -1,28 +1,20 @@
...
 
-    Flag
-        Enable a feature for specific users, groups, users meeting certain criteria (e.g. authenticated or staff),
-        or a certain percentage of visitors.
+#. Sign in to the LMS Django administration console for your base URL. For
+   example, ``http://{your_URL}/admin``.
 
-    Switch
-        Simple boolean, toggling a feature for all users.
+#. On the **Site Administration** page, locate **Credentials** and select **Program certificate configurations**
```
Comment (amangano-edx) [inline docs/features.rst]: Is this referring to the LMS Django-admin or the Credentials Django-admin?

Comment (amangano-edx) [reply docs/features.rst]: Line 8 makes it seem like you're referring to the LMS, but I think the section described by this line exists in Credentials.

File: docs/features.rst [hunk 3]
```diff
@@ -26,3 +26,24 @@ Permanent Feature Rollout
 Over time some features may become permanent and no longer need a feature gate around them. In such instances, the
 relevant code and tests should be updated to remove the feature gate. Once the code is released, the feature flag/switch
 should be deleted.
+
+Features
+=======================
+
+Enabling Hours of Effort
+~~~~~~~~~~~~~~~~~~~~~~~~
+To enable Hours of Effort, follow these steps.
```
Comment (amangano-edx) [inline docs/features.rst]: It might be good to add a sentence explaining what the 'Hours of Effort' is.

File: docs/configuring_certificates.rst [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+`Waffle documentation <http://waffle.readthedocs.org/en/latest/>`_.
+
+Permanent Feature Rollout
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Over time some features may become permanent and no longer need a feature gate around them. In such instances, the
+relevant code and tests should be updated to remove the feature gate. Once the code is released, the feature flag/switch
+should be deleted.
+
+
+Enabling Hours of Effort
```
Comment (catong) [inline docs/configuring_certificates.rst]: Suggest a title more like: "Displaying Hours of Effort on a Certificate"

File: docs/configuring_certificates.rst [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+------------------------
+Hours of Effort is a displayable field on a certificate. To enable Hours of Effort, follow these steps.
```
Comment (catong) [inline docs/configuring_certificates.rst]: "To enable Hours of Effort..." -> "To display the hours of effort for a program on the program certificate, follow these steps."

File: docs/configuring_certificates.rst [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+#. Sign in to the Credentials Django administration console for your base URL. For
+   example, ``http://{your_URL}/admin``.
+
+#. On the **Site Administration** page, locate **Credentials** and select **Program certificate configurations**
```
Comment (catong) [inline docs/configuring_certificates.rst]: Missing period at end of sentence.

File: docs/configuring_certificates.rst [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+#. Create or update the Program certificate you would like to enable Hours of Effort for.
```
Comment (catong) [inline docs/configuring_certificates.rst]: "Create or update the program certificate *configuration* for the certificate." (Is this right?)

File: docs/configuring_certificates.rst [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+#. Select the **Include hours of effort** checkbox.
+
+#. Select **Save**.
```
Comment (catong) [inline docs/configuring_certificates.rst]: Suggest adding a result step to clarify when the changes take effect. For example, do program certs have to be regenerated manually? e.g. "Program certificates generated after this change will include hours of effort."

File: docs/configuring_certificates.rst [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+To show Hours of Effort, the ``Total hours of effort`` field for a program in the course-discovery IDA must have a value.
```
Comment (catong) [inline docs/configuring_certificates.rst]: Recommend making this a note (``` .. note:: ```) and moving it earlier, immediately after "follow these steps."

File: docs/configuring_certificates.rst [hunk 10]
```diff
@@ -22,7 +29,27 @@ For information on creating or updating features, refer to the
...
 should be deleted.
+
+
+Enabling Hours of Effort
+------------------------
+To display the hours of effort for a program on the program certificate, follow these steps.
+
+.. note:: To show Hours of Effort, the ``Total hours of effort`` field for a program in the course-discovery IDA must have a value.
+
+#. Sign in to the Credentials Django administration console for your base URL. For
```
Comment (catong) [inline docs/configuring_certificates.rst]: Is this the *LMS* Django Administration site for your Credentials instance? and is the example URL accurate? Would be great if the form were consistent wherever we have it: see step 2 in "Internationalized Program Certificates": 
```Log in to the Django admin for your Credentials instance ({credentials_base_url}/admin).```

**Comment by catong** [comment]
@BenjiLee Just one additional comment about the "Log in to Django Admin" step

**Comment by BenjiLee** [comment]
@catong Ready for a final pass

File: docs/configuring_certificates.rst [hunk 11]
```diff
@@ -22,7 +29,26 @@ For information on creating or updating features, refer to the
...
 Over time some features may become permanent and no longer need a feature gate around them. In such instances, the
 relevant code and tests should be updated to remove the feature gate. Once the code is released, the feature flag/switch
 should be deleted.
+
+
+Enabling Hours of Effort
+------------------------
+To display the hours of effort for a program on the program certificate, follow these steps.
+
+.. note:: To show Hours of Effort, the ``Total hours of effort`` field for a program in the course-discovery IDA must have a value.
```
Comment (catong) [inline docs/configuring_certificates.rst]: @BenjiLee Can you provide more info about where this setting is? A file path or fila name? Is it done through Django Admin for the Discovery service?

You could convert the note into step 1: "Ensure that the "Total hours of effort" field in the blahblah settings file (located ...) has a value."

Comment (BenjiLee) [reply docs/configuring_certificates.rst]: To show Hours of Effort, the ``Total hours of effort`` field must have a value for a program set in the Discovery Django Admin at **Home** › **Course Metadata** › **Programs**. If this field has no value, Hours of Effort will not be displayed on the certificate.

**Comment by catong** [comment]
Belated 👍

**Pull request merged**